### PR TITLE
Updated  Assignments

### PR DESCRIPTION
**Pull request by kahyee**

### CODE REVIEW

**Comment by kahyee** [comment]
Completed Session 11 assignments

File: students/kahyee/Session11/comprehensions-top100.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+music = pd.read_csv("featuresdf.csv")
+
+possible_songs = [(x,y,z) for x, y, z in zip(music.name, music.danceability, music.loudness) if y > 0.8 and z < 5.0]
```
Comment (davidblaszka) [inline students/kahyee/Session11/comprehensions-top100.py]: should be -5.0 but otherwise great job!

File: students/kahyee/Session11/generator.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    intermediary = 0
+    
+    while True:
+        yield cur_value
+        intermediary = prev_value
+        prev_value = cur_value
+        cur_value += intermediary
+
+
+def isPrime(n):
```
Comment (davidblaszka) [inline students/kahyee/Session11/generator.py]: I like how you brought this out and made its own function. Would you be willing to show this to the class?

File: students/kahyee/Session11/iterator_1.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    """
+
+    def __init__(self, start, stop, step=1):
+        self.current = start - step
+        self.start = start
+        self.step = step
+        self.stop = stop
+
+    def __iter__(self):
+        self.current = self.start - self.step
```
Comment (davidblaszka) [inline students/kahyee/Session11/iterator_1.py]: Great job on this! Would you also be willing to talk about this to the class?

File: students/kahyee/Session11/comprehensions-top100.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+def dance_key(item):
+    return item[1]
+    
+sorted_songs = sorted(possible_songs, key= dance_key, reverse=True)
```
Comment (davidblaszka) [inline students/kahyee/Session11/comprehensions-top100.py]: nice

**Pull request merged**